### DISCUSSION TITLE
Eraser Cleanup

### DISCUSSION THREAD

**Discussion by crystalthoughts** [fabricjs/fabric.js#9916] [Q&A]
Hi, I notice that the bounds of a path stays the same after erasing. Is it possible to 'collapse' the operations down to a new path?

**Comment by asturur** [comment]
Hi @crystalthoughts the eraser path moved to a different repo https://github.com/ShaMan123/erase2d

**Comment by mhieupham1** [comment]
let see my example https://jsfiddle.net/mhieupham/4ztLa296/61/ @crystalthoughts

**Reply by crystalthoughts** [reply]
This doesn't collapse the bounding box to the new dims though?

**Comment by crystalthoughts** [comment]
Thank you!
